bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/edit.js

```diff
@@ -404,9 +404,25 @@ pimcore.object.helpers.edit = {
                             // apply tooltips
                             if(field.tooltip) {
                                 try {
+                                    var tooltipHtml = field.tooltip;
+
+                                    // special rule for classification store keys
+                                    // in this context there is no way to refer to the description of a classification store keys
+                                    // therefore the fallback tootip provided by the api will be separated and translated correctly
+                                    if (dataProvider.component !== undefined &&
+                                        dataProvider.component.bodyCls == 'pimcore_object_tag_classification_store' &&
```

Comment:
this looks a bit fragile - what if the bodyCls changes ... nobody will probably know to change the rule here as well.

Is there a reason why this cannot be done here?
https://github.com/pimcore/pimcore/blob/d07615a97ed4b95e4f4a1e82ed0354f82ed5b839/bundles/AdminBundle/Resources/public/js/pimcore/object/tags/classificationstore.js#L419

Reply:
That makes sense. I fixed it.